main.go

```diff
@@ -43,6 +58,10 @@ func main() {
 			Name:  "no-key, K",
 			Usage: "output without keys (and without color)",
 		},
+		cli.StringSliceFlag{
```

Comment:
Wow, StringSlice is now working, cool https://github.com/codegangsta/cli/pull/229
I wanted to use this for -k option too ...